tests/unit/unit-util.cpp

```diff
@@ -165,4 +241,9 @@ INSTANTIATE_TEST_SUITE_P(ParameterizedUnitUtilIntTests,
                          UnitUtilIntTests,
                          testing::ValuesIn(filter_solver_configurations({ TERMITER, THEORY_INT })));
 
+INSTANTIATE_TEST_SUITE_P(
+    ParameterizedUnitUtilDimacsTests,
+    UnitUtilDimacsTests,
+    testing::ValuesIn(filter_solver_configurations({ TERMITER, THEORY_INT })));
```

Comment:
I don't see a reason u would need support for `THEORY_INT` for this test. And in fact, possibly also no need for TERMITER. So maybe just use `available_solver_enums`.